src/CloudNative.CloudEvents.AspNetCore/CloudNative.CloudEvents.AspNetCore.csproj

```diff
@@ -8,9 +8,13 @@
     <PackageTags>cncf;cloudnative;cloudevents;events;aspnetcore;aspnet</PackageTags>
   </PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Http" Version="2.1.34" />
-    <PackageReference Include="System.Text.Encodings.Web" Version="6.0.0" Condition="'$(TargetFramework)'=='netstandard2.0' or '$(TargetFramework)'=='netstandard2.1'" />
+    <ItemGroup>

```

Comment:
Could all of these just be in a single ItemGroup? Alternatively, maybe create one ItemGroup for netstandard2.0/2.1, and one for everything else, so we don't need a condition on each element within it.

Reply:
I usually do item group as well but was following the existing packages. Will update tomorrow